apps/backoffice-v2/src/pages/Entity/components/CallToAction/CallToAction.tsx

```diff
@@ -45,23 +58,34 @@ export const CallToAction: FunctionComponent<ICallToActionProps> = ({ value, dat
     onCommentChange,
     noReasons,
     workflowLevelResolution,
-  } = useCallToActionLogic();
+    isReuploadResetable,
+  } = useCallToActionLogic({
+    contextUpdateMethod,
+    revisionReasons,
+    rejectionReasons,
+    onReuploadReset,
+  });
+
+  const handleDialogClose = useCallback(
+    (open: boolean) => {
+      if (!open) onDialogClose && onDialogClose();
```

Comment:
```suggestion
      if (!open && onDialogClose) onDialogClose();
```
Note this syntax should also work. 🙂 
```suggestion
      if (!open) onDialogClose?.();
```